new_relic_apdex_t/new_relic_apdex_t.php

```diff
@@ -30,27 +32,26 @@
  */
 function get_nr_connection_info( $env = 'dev' ) {
   $output = array();
-  $req    = pantheon_curl( 'https://api.live.getpantheon.com/sites/self/bindings?type=newrelic', null, 8443 );
-  $meta   = json_decode( $req['body'], true );
-
-  foreach ( $meta as $data ) {
-    if ( $data['environment'] === $env ) {
-      if ( empty( $data['api_key'] ) ) {
-        echo "Failed to get API Key\n";
+  $req    = pantheon_curl( 'https://api.live.getpantheon.com/sites/self/name', null, 8443 );
```

Comment:
We shouldn't use `pantheon_curl` in sample code. That mistake was first made back in 2016, but if the only thing we need is the site name, that's available in `$_ENV('PANTHEON_SITE_NAME')`.